lib/rules/brace-style.js

```diff
@@ -0,0 +1,193 @@
+/**
+ * @fileoverview Rule to flag block statements that do not use the one true brace style
+ * @author Ian Christian Myers
+ */
```

Comment:
Need to add a header like https://github.com/silvermine/eslint-plugin-silvermine/blob/696a63ae15932a45fbd0562f10eef0842f309aba/lib/rules/indent.js